lib/asciidoctor/iso/term_lookup_cleanup.rb

```diff
@@ -28,22 +28,31 @@ def set_termxref_tags_target
         xmldoc.xpath('//termxref').each do |node|
           target = normalize_ref_id(node.text)
           if termlookup[target].nil?
-            log.add('AsciiDoc Input', node, "#{target} does not refer to a real term")
+            remove_missing_ref(node, target)
             next
           end
 
           modify_ref_node(node, target)
         end
       end
 
+      def remove_missing_ref(node, target)
+        warn(%Q(Error: Term reference in `term[#{target}]` missing: \
+                "#{target}" is not defined in document.).gsub(/\s+/, ' '))
+        log.add('AsciiDoc Input', node, "#{target} does not refer to a real term")
+        node.next.remove
+        term_name_node = node.previous.previous
+        term_name_node.remove
+        node.previous.remove
+        node.add_previous_sibling(term_name_node.text)
+        node.remove
+      end
+
```

Comment:
So, you're converting `<em>X</em> <termxref>Y</termxref>` to just `X` if the id is missing? If that was the explicit instruction from @ronaldtse, OK...

Reply:
Yes, @ronaldtse want to render it as as a text. @ronaldtse is it correct?

Reply:
Yes this is correct. We do not want to render broken references.